src/taco-dependency-installer/dependencyInstaller.ts

```diff
@@ -355,14 +372,14 @@ module TacoDependencyInstaller {
 
             return installerUtils.promptUser(resources.getString("YesExampleString"))
                 .then(function (answer: string): Q.Promise<any> {
-                    if (answer === resources.getString("YesString")) {
+                    if (answer.toLocaleLowerCase() === resources.getString("YesString")) {
```

Comment:
Yep, I think that would be the best solution... I think that it should probably be part of the same changeset that will modify the telemetry prompt though, so for now I will just make this quick change to at least make the prompt similar to what we have elsewhere